cpp/csp/python/PyStructToJson.cpp

```diff
@@ -72,7 +72,7 @@ inline rapidjson::Value toJson( const TimeDelta& val, const CspType& typ, rapidj
     // Convert TimeDelta to <sign><seconds>.<microseconds>
     // sign( 1 ) + seconds ( 18 ) + '.'( 1 ) + microseconds( 9 ) + '\0'( 1 )
     char buf[32] = {};
-    auto seconds = val.abs().asSeconds();
+    long seconds = val.abs().asSeconds();
```

Comment:
curious what this warning was?

Reply:
https://github.com/Point72/csp/actions/runs/9896204748/job/27337875177?pr=321#step:8:2647